### PR TITLE
Fix minor corrections

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

[x] Update README.md config more simple.
[x] Fix dependencies composer.json.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -79,14 +79,19 @@ jobs:
...
+        run: composer update --ignore-platform-reqs --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
 
-      - name: Run redis tests with phpunit
-        run: vendor/bin/phpunit --coverage-clover=coverage.clover
+      - name: Run tests with phpunit and coverage on Linux php 7.4
+        if: matrix.os == 'ubuntu-latest' && matrix.php-version == '7.4'
+        run: vendor/bin/phpunit --coverage-clover=coverage.clover --colors=always
+
+      - name: Run tests with phpunit without coverage on Windows php 7.4 - 8.0 and Linux php 8.0
+        if: matrix.os == 'windows-latest' || (matrix.os == 'ubuntu-latest' && matrix.php-version == '8.0')
```
Comment (samdark) [inline .github/workflows/build.yml:93]: See https://github.com/yiisoft/yii-bootstrap4/pull/34

File: README.md [hunk 2]
```diff
@@ -31,74 +31,28 @@ composer require yiisoft/db-redis
...
-        return $connection;
-    }
+    RedisConnection::class => [
+        '__class' => RedisConnection::class,
+        '__construct()' => [
+            Reference::to(EventDispatcherInterface::class),
+            Reference::to(LoggerInterface::class),
+        ],
+        'hostname()' => [$params['yiisoft/db-redis']['dsn']['host']],
+        'port()' => [$params['yiisoft/db-redis']['dsn']['port'])],
```
Comment (samdark) [inline README.md:52]:
```suggestion
        'port()' => [$params['yiisoft/db-redis']['dsn']['port']],
```

File: composer.json [hunk 3]
```diff
@@ -19,18 +19,18 @@
         "ext-json": "*",
         "yiisoft/arrays": "^3.0@dev",
         "yiisoft/db": "^3.0@dev",
-        "yiisoft/strings": "^1.0"
+        "yiisoft/log": "^3.0@dev",
+        "yiisoft/log-target-file": "^3.0@dev",
```
Comment (samdark) [inline composer.json:23]: Are you sure specific target is required for the package?

**Review by samdark — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (terabytesoftw) [reply .github/workflows/build.yml]: Fixed.

File: composer.json (hunk 3 above)
Comment (terabytesoftw) [reply composer.json:23]: no.

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md:52]: fixed.

**Pull request merged**